api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -167,44 +179,48 @@ def get_item(self, api_key: str) -> dict:
 
 
 class DynamoEnvironmentV2Wrapper(BaseDynamoEnvironmentWrapper):
-    table_name = settings.ENVIRONMENTS_V2_TABLE_NAME_DYNAMO
-
-    ENVIRONMENT_ID_ATTRIBUTE = "environment_id"
-    DOCUMENT_KEY_ATTRIBUTE = "document_key"
-    ENVIRONMENT_API_KEY_ATTRIBUTE = "environment_api_key"
-    ENVIRONMENT_API_KEY_INDEX_NAME = "environment_api_key-index"
-
-    def get_environment_by_api_key(self, environment_api_key: str) -> dict:
-        get_item_kwargs = {
-            "IndexName": self.ENVIRONMENT_API_KEY_INDEX_NAME,
-            "Key": {
-                self.ENVIRONMENT_API_KEY_ATTRIBUTE: environment_api_key,
-                self.DOCUMENT_KEY_ATTRIBUTE: "META",
-            },
-        }
-        try:
-            return self._table.get_item(**get_item_kwargs)["Item"]
-        except IndexError:
-            raise ObjectDoesNotExist()
-
-    def get_identity_overrides(
-        self, environment_id: int, feature_id: int = None
-    ) -> typing.List[dict]:  # TODO better typing?
-        document_key_begins_with = "identity_override"
-        if feature_id:
-            document_key_begins_with += f":{feature_id}"
-        key_expression_condition = Key(self.ENVIRONMENT_ID_ATTRIBUTE).eq(
-            environment_id
-        ) & Key(self.DOCUMENT_KEY_ATTRIBUTE).begins_with(document_key_begins_with)
+    def get_table_name(self):
```

Comment:
@matthewelwell Various improvements here that will introduce conflicts in your PR:
- Constants live in their own module now.
- `get_identity_overrides` is now `get_identity_overrides_by_feature_id` to avoid dead code paths — to my best knowledge, only Edge API will need to fetch overrides by environment id alone.